CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 Changes by Version
 ==================
 
+0.19.0 (2021-01-27)
+-------------------
+* Bumped OpenTelemetry Collector to v0.19.0 ([#nn](https://github.com/open-telemetry/opentelemetry-operator/pull/nn), [@jpkrohling](https://github.com/jpkrohling))
```

Comment:
nit: Does it need to refer to itself? Maybe in these circumstances it should just have a line that says "Bumping version of OpenTelemetry Collector to v0.19.0 - no other changes"